psnineplus.js

```diff
@@ -2286,39 +2351,107 @@
+                                        count_str = `<b>${this.y}</b>`;
+                                    else
+                                        count_str = `<b>无评论</b>`;
+                                    return week_str + count_str;
+                                }
+                            }
+                        }
                     ];
                     // 标题设置
-                    const scoreTrendTitle = {
-                        text: '均分走势',
-                        style: {
-                            color: '#808080',
-                        },
-                    };
+                    const scoreTrendTitle = [
+                        {
+                            text: '均分走势',
+                            style: {
+                                color: '#808080',
+                            }
+                        }, {
+                            text: '热度走势',
+                            style: {
+                                color: '#808080',
+                            }
+                        }
+                    ];
+                    let comment_min_count = Number.MAX_SAFE_INTEGER, comment_max_count = Number.MIN_SAFE_INTEGER;
+                    comment_trend.forEach((comment_count) => {
+                        if (comment_count[1] > comment_max_count)
+                            comment_max_count = comment_count[1];
+                        if (comment_count[1] < comment_min_count)
+                            comment_min_count = comment_count[1];
+                    });
                     // Y轴设置
-                    const scoreTrendYAxis = {
-                        title: {
-                            text: '平均分',
-                        },
-                        min: min_score - 0.2 > 0 ? min_score - 0.2 : min_score,
-                        max: max_score + 0.2 < 10 ? max_score + 0.2 : 10,
-                        endOnTick: true,
-                        tickInterval: 0.1
-                    };
+                    const scoreTrendYAxis = [
+                        {
+                            title: {
+                                text: '平均分',
+                                style: {
+                                    color: '#7CB5EC',
+                                }
+                            },
+                            min: min_score - 0.2 > 0 ? min_score - 0.2 : min_score,
+                            max: max_score + 0.2 < 10 ? max_score + 0.2 : 10,
+                            endOnTick: true,
+                            tickInterval: 0.1,
+                            opposite: false
+                        }, {
+                            title: {
+                                text: '周评分数',
+                                style: {
+                                    color: '#F28D8F',
+                                }
+                            },
+                            min: comment_min_count,
```

Comment:
```suggestion
                            min: Math.min.apply(Math, comment_trend.map((i) => {return i[1];})),
```
或许可以考虑一下用这种方式减少一些代码量，这样`2411-2417` 这部分都可以删除了（请随意）